test-app/src/components/debugger.tsx

```diff
@@ -310,7 +311,7 @@ export const Debugger = () => {
 
       <Box>
         <ButtonGroup spacing={4} my="base">
-          <Button mt={3} onClick={callFaker}>
+          <Button data-test={TransactionsSelectors.ContractCall} mt={3} onClick={callFaker}>
```

Comment:
```suggestion
          <Button data-test={TransactionsSelectors.BtnCallContract} mt={3} onClick={callFaker}>
```
Suggested better selector name. This or something more action-based, verby would be clearer.